modules/chttrans/chttrans.cpp

```diff
@@ -213,6 +213,23 @@ void Chttrans::save() {
     safeSaveAsIni(config_, "conf/chttrans.conf");
 }
 
+const Configuration *Chttrans::getConfig() const {
+#ifdef ENABLE_OPENCC
+    std::vector<std::string> profiles;
+    StandardPath::global().scanFiles(
+        StandardPath::Type::Data, "opencc",
+        [&profiles](const std::string &path, const std::string &, bool) {
+            if (stringutils::endsWith(path, ".json")) {
+                profiles.emplace_back(path);
+            }
+            return true;
+        });
+    config_.openCCS2TProfile.annotation().setProfiles(profiles);
```

Comment:
Add a place holder entry for default, since the real default comes from a OpenCC defined macro.

Reply:
If you want to change a different profile as default, make sure you check s2tw.json does exist at compile time with CMake.